Feat: Notary Website Redesign

thisisobate opened the pull request:
**Todos**
- [x] Flesh out hero section
- [x] [Style the buttons](https://github.com/notaryproject/notaryproject.dev/issues/112) - assigned to Hamees
- [x] [Create feature card component](https://github.com/notaryproject/notaryproject.dev/issues/113) - assigned to Hamees
- [x] Create layout/structure using plain HTML
- [x] Style page
- [x] Implement mobile responsiveness
- [x] Convert HTML to Hugo templates
- [x] Update the landing page based on the latest change
- [x] Update content with real data
- [x] Add two links to two buttons
- [x] Move the logo to corner at the top left
- [x] Change Resources to Blog for the first version
- [x] Pull the updates from the latest main branch for /doc
- [x] Clean code
  - [x] fix font sizing
  - [x] remove redundant code
  - [x] maintain consistent units

**Follow-up PR**
- [ ] Make the “News & Blogs” section to be dynamic
- [ ] Split file into components

FeynmanZhou commented:
As discussed with @thisisobate , we will start to review and merge it into the main branch this week. @thisisobate Could you please change its status from WIP to Ready for review after you finalize the development in your TODO list? Thanks!

thisisobate commented:
Sure! Will do! @FeynmanZhou

FeynmanZhou commented:
Hi @thisisobate ,

1. Could you pls change the button from `Demo` to `GitHub`
2. Change `you` to `your`
3. Shall we add the 4th user case "Zot" in this PR or in a follow-up PR?
4. The font color is missing on the community page
![image](https://user-images.githubusercontent.com/40452856/219645389-25555cdf-b4ee-4b43-9e5c-81c0ed6e61c2.png)

![image](https://user-images.githubusercontent.com/40452856/219645846-29fdf7ba-a624-4293-b836-07bd709cfe79.png)

5. The documentation content needs to rebase and sync with the https://notaryproject.dev/docs/
6. The code block style is not correct, it has an unexpected blue margin
![image](https://user-images.githubusercontent.com/40452856/219646596-10f480c5-c694-479e-842d-76491e82711c.png)

yizha1 commented:
Page settings: Zoom 100%

1. Seems the title and the content are mixed in one paragraph, so it read a bit weird "Notary and Notation Notary".
![image](https://user-images.githubusercontent.com/107919912/219649080-6e3c80be-3d8c-458f-95e8-9e0b437b7a10.png)

2.  There is no content preview on the second blog
![image](https://user-images.githubusercontent.com/107919912/219649296-d04f8949-a1cd-4c96-ab61-a9884d176a18.png)

3. The logo is too small
![image](https://user-images.githubusercontent.com/107919912/219649373-bceb59d7-217f-41ac-99a8-82bbed9d99e5.png)

4. the words were not showed completely on one line
![image](https://user-images.githubusercontent.com/107919912/219649437-e34d50e0-35d8-4837-9fa0-f8fef8f202db.png)

5. Seems the left and right are too far from each other
![image](https://user-images.githubusercontent.com/107919912/219649522-9c55c5c9-e154-4a01-95e8-a5ff21272280.png)

6. Is this the right logo for AWS web service? If not, we should fix it before publishing
![image](https://user-images.githubusercontent.com/107919912/219649661-d5607a82-db1d-46ac-bc52-496d113c30e8.png)

yizha1 reviewed (commented):
See comments above

thisisobate commented:
@FeynmanZhou I just addressed all your comments in my recent commits.

> The documentation content is outdated, please rebase and sync with the main branch (https://notaryproject.dev/docs/)

I already did that in a previous commit. [See here](https://github.com/notaryproject/notaryproject.dev/pull/110/commits/b5314de5a4157602418db0aca8645719840d514a)

> Change the AWS logo (Please replace it with a new one that I provided on the Figma)

The AWS logo in the deploy preview is same as what we have on figma. Can you please check and confirm?

> If I add a new blog, will it be automatically displayed on the landing page? If the blog quantity is more than three, will it only display the latest three here?

This is exactly what we would be tackling in a follow-up PR.

thisisobate commented:
@yizha1 
> Seems the title and the content are mixed in one paragraph, so it read a bit weird "Notary and Notation Notary"

Yeah, it's an expected behaviour. By default, page summary is auto-generated by Hugo. It picks the first few lines in the document and displays it as summary. 

> There is no content preview on the second blog
This is done by design. You can check the Figma file to see what it looks like.

> The logo is too small

I already improved this behavior in my previous commit. The logos are expected to shrink as soon as you minimize the viewport (window/screen). The reason for this is to prevent content overflow. The orientation changes from row to column once it switches to a mobile viewport so it's not a huge problem, especially for mobile users.

> the words were not showed completely on one line

I improved this behavior too. Like I mentioned earlier, contents are shrunk to fit within their current viewport. It's best to have content wrap than to cause an overflow. 

> Seems the left and right are too far from each other
I don't think so. Whether small or large screens, the space between the two items is shrunk to prevent too much whitespace.

> Is this the right logo for AWS web service? If not, we should fix it before publishing

I think it is.

FeynmanZhou commented:
@thisisobate 
Thanks for your continuous support and addressing the comments. There are last a few suggested changes:

1. The documentation content is still outdated after I checked the same page between [preview](https://deploy-preview-110--notarydev.netlify.app/docs/installation/cli/#verify-the-notation-cli-checksum) and [production](https://notaryproject.dev/docs/installation/cli/#verify-the-notation-cli-checksum). You can also scroll down to see the last modified date for each doc. Could you please re-check this one?
2. Support center alightment. Currently, it still displays strange on my 13' inch screen. See a good example 
![image](https://user-images.githubusercontent.com/40452856/219873420-68279077-5756-49b4-bbe5-4c63bce952c3.png)
3. AWS logo needs to be replaced with a new one. 
![image](https://user-images.githubusercontent.com/40452856/219873629-0d458120-a1b6-48df-a791-0076c92d6b13.png)

Thanks

yizha1 reviewed (approved):
LGTM Thanks @thisisobate for your contribution!

FeynmanZhou reviewed (approved):
LGTM. We are ready to merge it and push the new website to production.

yizha1 reviewed (approved):
LGTM. woot~

vaninrao10 reviewed (approved):
Please remove "Signer" word from the AWS section. It should be only AWS. You can open a new PR and take care of this comment. Rest contents LGTM.

priteshbandi reviewed (dismissed):
LGTM

toddysm commented on content/en/_index.html:
```diff
@@ -13,30 +13,322 @@
+    </div>
+    <div class="text-container text-left">
+      <p>
+        AWS Signer team is using and contributing to Notation, building the
```
@vaninrao10 requested that we remove the Signer from the text.

FeynmanZhou replied:
@toddysm As Vani mentioned above, this will be addressed in a follow-up PR. I will raise a PR to update the description. Let's agree on merging Uche's change first.

thisisobate replied:
I already pushed a fix for this.
I'm trying to fix the issue of commits not being verified.

toddysm commented on content/en/_index.html:
```diff
@@ -13,30 +13,322 @@
+    </div>
+    <div class="text-container text-left">
+      <p>
+        Zot registry supports store Notation signature as OCI artifacts
```
Do we have confirmation from the Zot team that we can use their name?

FeynmanZhou replied:
Yes. We confirmed with Ramkumar from the Zot team in a community call.

toddysm commented on content/en/_index.html:
```diff
@@ -13,30 +13,322 @@
+      <img src="aws_logo.svg" alt="AWS logo" />
+    </div>
+    <div class="img-container-mdl is-centered">
+      <img src="cncf_logo.svg" alt="CNCF logo" />
```
I am not sure how "In collaboration with CNCF" sounds in this context. Notary is a CNCF project but I think we should list only collaborators.

FeynmanZhou replied:
This list includes major contributor and collaborator organizations. It's fine to add CNCF's logo to the contributor org list since CNCF has employees who are contributing to Notary

iamsamirzon commented:
@toddysm , @FeynmanZhou - I have more than a few comments, mostly nits. What is the best way to provide that? I can't seem to provide inline comments in the https://deploy-preview-110--notarydev.netlify.app/ 
cc: @vaninrao10

FeynmanZhou commented:
> @toddysm , @FeynmanZhou - I have more than a few comments, mostly nits. What is the best way to provide that? I can't seem to provide inline comments in the https://deploy-preview-110--notarydev.netlify.app/ cc: @vaninrao10

@iamsamirzon Could you screenshot your suggested changes and paste them here?

thisisobate commented:
Couldn't make commits to be verifiable in this PR so I decided to create a new one.
Closed in favor of #139
